src/app/action-reducer-map.ts

```diff
@@ -1,6 +1,15 @@
-import { ActionReducerMap } from '@ngrx/store';
+import {ActionReducerMap} from '@ngrx/store';
+import {notificationReducer} from './common/states/reducers/notification.reducer';
+import {Route} from './common/models/route';
+import { TripPlannerState } from './passengers/actions/trip-planner.action';
```

Comment:
Not sure what the standard is for this but be consistent on whether you are including spaces or not including spaces